swarm:
   formation: 'box' #no, box
   robots: 2 #used to "no" formation
   map: 'box' #box,corridor,empty
   VonMisesKappa: 0
   VonMisesMu: 0